app/views/layouts/_google_analytics.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if Rails.env.production? %>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-163725354-1"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+  
+    gtag('config', 'UA-163725354-1');
```

Comment:
should we hide our UA token?

Reply:
Sorry for the late reply. I can't add the UA token to the `config/credentials.yml` file because I'm not able to decrypt it.

Is there another way that we can store this?